install_config/persistent_storage/topics/glusterfs_example_registry.adoc

```diff
@@ -0,0 +1,46 @@
+. In your inventory file, set the following variable under `[OSEv3:vars]`:
++
+----
+[OSEv3:vars]
+...
+openshift_hosted_registry_storage_kind=glusterfs
+----
+
+. Add `glusterfs_registry` in the `[OSEv3:children]` section to enable the
+`[glusterfs_registry]` group:
++
+----
+[OSEv3:children]
+masters
+nodes
+glusterfs_registry
+----
+
+. Add a `[glusterfs_registry]` section with entries for each storage node that
+will host the GlusterFS storage. For each node, set `glusterfs_devices` to a
+list of raw block devices that will be completely managed as part of a
+GlusterFS cluster. There must be at least one device listed. Each device must
+be bare, with no partitions or LVM PVs. Specifying the variable takes the form:
++
+----
+<hostname_or_ip> glusterfs_devices='[ "</path/to/device1/>", "</path/to/device2>", ... ]'
+----
++
+For example:
++
+----
+[glusterfs_registry]
+node11.example.com glusterfs_devices='[ "/dev/xvdc", "/dev/xvdd" ]'
```

Comment:
Should this be added to end of the 3 lines - openshift_node_labels="{'role': 'infra'}"